TODO.md

```diff
@@ -0,0 +1,7 @@
+# TODO
+
+This file tracks the list of long term wants and needs for the RStudio image
+
+- [ ] /etc/rstudio/rserver.conf needs auth-none=1 adding
```

Comment:
You've done that here, haven't you?: https://github.com/ministryofjustice/analytics-platform-rstudio/pull/67/files#diff-3254677a7917c6c01f55212f86c57fbfR132

Reply:
Reverted. I was just testing a few things out with the S6 overlay. Prize catch 🎉